txcache/trackedBlock.go

```diff
@@ -31,6 +31,41 @@ func newTrackedBlock(
 	}
 }
 
+func (tb *trackedBlock) createOrUpdateVirtualRecords(
+	session SelectionSession,
+	skippedSenders map[string]struct{},
+	sendersInContinuityWithSessionNonce map[string]struct{},
+	accountPreviousBreadcrumb map[string]*accountBreadcrumb,
+	virtualAccountsByAddress map[string]*virtualAccountRecord,
+) error {
+	for address, breadcrumb := range tb.breadcrumbsByAddress {
+		_, ok := skippedSenders[address]
+		if ok {
+			continue
+		}
+
+		accountState, err := session.GetAccountState([]byte(address))
+		if err != nil {
+			log.Debug("selectionTracker.createVirtualSelectionSession",
+				"err", err)
+			return err
+		}
+
+		accountNonce := accountState.GetNonce()
+
+		if !breadcrumb.isContinuous(address, accountNonce,
```

Comment:
Check fails if this is the second breadcrumb (second tracked block in the chain of tracked blocks)?

Oh, or maybe you handle the non-first-breadcrumb case within `isContinuous`? (I'll read along and see).